prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+CP1404 Practical 5
+Emails
+Estimate: 0.5 seconds
+Actual: 0.5 seconds
+"""
+
+
+def main():
+    """Stores users emails(keys) and names(values) into a dictionary"""
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = get_name_from_email(email)
+        answer = input(f"Is your name {name}: ").upper()
+        if answer != "Y" and answer != "":
+            name = input("Name: ")
+        email_to_name[email] = name
+        email = input("Email: ")
+    for email, name in email_to_name.items():
+        print(f"{name} ({email})")
+
+
+def get_name_from_email(email):
+    """Determine possible name from email"""
+    first_part = email.split('@')[0]
```

Comment:
The variable name "first_part" seems a bit confusing to me. Would it be better to use a name like "prefix"?

Reply:
I agree